modules/exploits/windows/browser/ntr_activex_check_bof.rb

```diff
@@ -51,6 +51,7 @@ def initialize(info = {})
       'DefaultOptions' =>
         {
           'EXITFUNC' => 'process',
```

Comment:
Do a few of these change behavior? Assuming the hash keys are reinitialized on the second assignment, was 'EXITFUNC' undefined originally, or was it set to 'process'. Does it make a difference?